### PR TITLE
Show status in csv format.

### PR DESCRIPTION
**Pull request by amjith**
## Description
Show error status in CSV output format.

## Checklist

- [X] I've added this contribution to the `changelog.rst`.

### CODE REVIEW

**Comment by j-bennet** [comment]
@amjith I thought we aren't displaying status in CSV [intentionally](https://github.com/dbcli/pgcli/pull/1173), because it was breaking `pspg` pager. Can you explain why we need this now?

File: changelog.rst [hunk 1]
```diff
@@ -4,7 +4,9 @@
 Bug fixes:
 ----------
 
-* Fix the bug with Redshift not displaying word count in status ([related issue](https://github.com/dbcli/pgcli/issues/1320)).
+* Fix the bug with Redshift not displaying wor count in status ([related issue](https://github.com/dbcli/pgcli/issues/1320)).
```
Comment (lelit) [inline changelog.rst:7]: s/wor/word/?

**Comment by amjith** [comment]
I believe [pspg](https://github.com/okbob/pspg) implemented a special flag to handle pgcli. `--pgcli-fix`

**Comment by lelit** [comment]
> So what happens now with `pspg`

FWIW, I'm happy with minimal configuration suggested at the end of the [pgcli](https://github.com/okbob/pspg#pgcli) section:
```
pager = /usr/bin/pspg --rr=2 --quit-if-one-screen --ignore-case
```


> I believe pspg implemented a special flag to handle pgcli: `--pgcli-fix`

Yes, but AFAICT that targets problems when using `CSV` output. Using the recommended `table_format = psql` it's not needed.

**Comment by j-bennet** [comment]
@amjith I noticed that in this comment:

https://github.com/dbcli/pgcli/issues/1326#issuecomment-1162472608

the author says that even with this cnange, `pgcli` still doesn't behave the right way when displaying stderr, while litecli does. Do you know what is different?

**Pull request merged**